exporter/zipkinexporter/zipkin.go

```diff
@@ -188,19 +257,23 @@ func (ze *zipkinExporter) ConsumeTraceData(ctx context.Context, td consumerdata.
 	}()
 
 	goodSpans := 0
-	for _, span := range td.Spans {
-		sd, err := spandatatranslator.ProtoSpanToOCSpanData(span)
-		if err != nil {
-			return consumererror.Permanent(err)
+	/*
```

Comment:
when the method is gone this won't be an issue but nit: delete commented code :)